lib/rules/block-indentation.js

```diff
@@ -393,8 +393,8 @@ export default class BlockIndentation extends Rule {
         this.log({
           message: warning,
           node,
-          line: node.loc.end.line,
-          column: node.loc.end.column,
+          line: node.loc.start.line,
+          column: node.loc.start.column,
```

Comment:
Quick question - could you just remove these two lines? I think by default it should try to take the `line` and `column` from `node.loc.start`

Reply:
Thanks so much for calling this out @lin-ll! Yes removing the two lines also resolves the reporting error and in a cleaner way too.